litestar/cli/_utils.py

```diff
@@ -10,15 +10,30 @@
 from pathlib import Path
 from typing import TYPE_CHECKING, Any, Callable, Generator, Iterable, Sequence, TypeVar, cast
 
-from click import ClickException, Command, Context, Group, pass_context, style
-from rich.console import Console
+from rich import get_console
 from rich.table import Table
 from typing_extensions import Concatenate, ParamSpec
 
 from litestar import Litestar, __version__
 from litestar.middleware import DefineMiddleware
 from litestar.utils import get_name
 
+rich_click_installed = False
+try:
+    import rich_click  # noqa: F401
+
+    rich_click_installed = True
+except ImportError:
+    pass
+
+if TYPE_CHECKING or not rich_click_installed:
```

Comment:
```suggestion
RICH_CLICK_INSTALLED = False
try:
    import rich_click  # noqa: F401

    rich_click_installed = True
except ImportError:
    pass

if TYPE_CHECKING or not RICH_CLICK_INSTALLED:
```

Reply:
If we intend to use this outside this module it should be uppercase to indicate a constant.